plugin/storage/integration/integration_test.go

```diff
@@ -149,14 +149,18 @@ func (s *StorageIntegration) testGetLargeSpan(t *testing.T) {
 func (s *StorageIntegration) testGetOperations(t *testing.T) {
 	defer s.cleanUp(t)
 
-	expected := []string{"example-operation-1", "example-operation-3", "example-operation-4"}
+	expected := []spanstore.Operation{
+		{Name: "example-operation-1"},
+		{Name: "example-operation-3", SpanKind: "server"},
+		{Name: "example-operation-4", SpanKind: "client"}}
```

Comment:
ES tests are failing, as I was expecting (but Badger tests do not, which is unexpected)
```
        integration_test.go:168: 
            	Error Trace:	integration_test.go:168
            	Error:      	Should be true
            	Test:       	TestElasticsearchStorage_AllTagsAsObjectFields/GetOperations
        integration_test.go:169: 	 Expected: [{example-operation-1 } {example-operation-3 server} {example-operation-4 client}]
        integration_test.go:170: 	 Actual  : [{example-operation-1 } {example-operation-3 } {example-operation-4 }]
```

I think you may need to introduce some "capability" flag into StorageIntegration so that the storage backend can declare whether it supports returning spanKind or not, and then the test should perform different expectation asserts depending on that flag.